pyalgotrading/algobulls/connection.py

```diff
@@ -462,10 +478,23 @@ def start_job(self, strategy=None, start=None, end=None, instruments=None, lots=
                 'paramValue': parameters[parameter_name]
             })
 
+        # get exchange location
+        exchange = 'NSE'
+        location = SEGMENT_COUNTRY_MAP[exchange]
+
+        if len(instruments[0].split(':')) == 2:
+            exchange = instruments[0].split(':')[0]
+            if SEGMENT_COUNTRY_MAP.get(exchange) is not None:
+                location = SEGMENT_COUNTRY_MAP.get(exchange)
```

Comment:
this `if else` can be reduced to a single `if` without an `else`

Reply:
Not Done : else part is required if we get have issue inside the second if